manifests/instance.pp

```diff
@@ -355,7 +355,81 @@
     owner   => $config_owner,
     group   => $config_group,
     mode    => $config_file_mode,
-    content => template($conf_template),
+    content => epp(
+      $conf_template,
+      {
+        daemonize                     => $daemonize,
+        pid_file                      => $pid_file,
+        protected_mode                => $protected_mode,
+        port                          => $port,
+        tcp_backlog                   => $tcp_backlog,
+        bind_arr                      => $bind_arr,
+        unixsocket                    => $unixsocket,
+        unixsocketperm                => $unixsocketperm,
+        timeout                       => $timeout,
+        tcp_keepalive                 => $tcp_keepalive,
+        log_level                     => $log_level,
+        _real_log_file                => $_real_log_file,
```

Comment:
Do we need to pass this as `_real_log_file`?
```suggestion
        log_file                => $_real_log_file,
```

Reply:
Yes, absolutely.  
Fixed.